### ISSUE TITLE
Unable to reproduce the results of sparse conv on modelnet40

### ISSUE THREAD

**Issue by bollossom** [Colin97/OpenShape_code#38]
The result of training for 200 epochs using 8 A800 GPUs on the ModelNet40 dataset is only 80.06% accuracy. All other parameters are the same as those in the train.yaml configuration file.

**Comment by eliphatfs** [comment]
I think the result is reasonable as this is only differing by 3% to our checkpoint. There are still many random factors leaving behind, like data sampling. Also, I think we used fewer GPUs so the effective batch size differ, which might lead to this discrepancy. You will need to tune the batch size or the learning rate schedule for better performance at 8 GPUs. If you kept many checkpoints you can try a self-ensemble which reduces these effects.

**Comment by bollossom** [comment]
Thanks for your reply. Could you tell me some other training hyper parameters, such as the batch size of the learning system and whether to warm up? I think 1000 epochs is too much, and whether to experiment with cosine decay?

**Comment by bollossom** [comment]
I think I have figured out the reason. I chose the 5.3M parameter version of the SparseConv model, which achieved an 80.4% result on ModelNet40. The original paper achieved an 83.4% zero-shot effect on ModelNet40, which likely used the model with 41.3M parameters.
